src/app/subject/store/sensors-data/sensors-data.effects.ts

```diff
@@ -66,15 +67,23 @@ export class SensorsDataEffects {
   @Effect()
   updateTimeInterval$ = this.actions$.pipe(
     ofType(actions.SET_TIME_FRAME),
-    withLatestFrom(this.store.select(fromSubject.getSensorsDataTimeFrame)),
+    debounceTime(1000),
```

Comment:
what happens if we don't debouceTime here?

Reply:
It will keep making requests when the brush is updated even when a response hasn't been received yet. This is to limit the requests sent.